react/features/recording/middleware.js

```diff
@@ -131,29 +132,27 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => async action =>
     }
 
     case START_LOCAL_RECORDING: {
-        if (!LocalRecordingManager.isRecordingLocally()) {
-            try {
-                await LocalRecordingManager.startLocalRecording({ dispatch,
-                    getState });
-                const props = {
-                    descriptionKey: 'recording.on',
-                    titleKey: 'dialog.recording'
-                };
-
-                dispatch(playSound(RECORDING_ON_SOUND_ID));
-                dispatch(showNotification(props, NOTIFICATION_TIMEOUT_TYPE.MEDIUM));
-                dispatch(updateLocalRecordingStatus(true));
-            } catch (err) {
-                console.error('Capture failed', err);
-
-                const noTabError = err.message === 'WrongSurfaceSelected';
-                const props = {
-                    descriptionKey: noTabError ? 'recording.surfaceError' : 'recording.error',
-                    titleKey: 'recording.failedToStart'
-                };
-
-                dispatch(showErrorNotification(props, NOTIFICATION_TIMEOUT_TYPE.MEDIUM));
-            }
+        try {
+            await LocalRecordingManager.startLocalRecording({ dispatch,
+                getState });
```

Comment:
doesn't this fit in 1 line?

Reply:
It does, but it breaks another rule 
![image](https://user-images.githubusercontent.com/11474153/171815420-a7b29815-eeaf-4bb5-a305-35907ae9f731.png)

Reply:
ah that one